docs/cdp/cdp_activation/cdp_configuration.md

```diff
@@ -0,0 +1,65 @@
+---
+description: Step-by-step configuration procedure of Ibexa CDP.
+---
+
+# Configuration
+
+To configure [[= product_name_cdp =]], use the `ibexa.system.<scope>.cdp` [configuration key](configuration.md#configuration-files):
+
+```yaml
+ibexa:
+    system:
+        default:
+            cdp:
+                account_number: 123456
+                data_export:
+                    user_data:
+                        transport: stream_file
+                        stream_file:
+                            stream_id: 00000000-00000000-00000000-00000000
+                    content_data:
+                        transport: stream_file
+                        stream_file:
+                            stream_id: 00000000-00000000-00000000-00000000
+                    product_data:
+                        transport: stream_file
+                        stream_file:
+                            stream_id: 00000000-00000000-00000000-00000000
+                activations:
+                        client_id: '%env(CDP_ACTIVATION_CLIENT_ID)%'
+                        client_secret: '%env(CDP_ACTIVATION_CLIENT_SECRET)%'
+                        segment_group_identifier: ibexa_cdp
+```
+
+- **Account number** - can be obtained from Accounts settings in [[= product_name_cdp =]] dashboard
+- **Stream ID** - use Stream ID generated during data import from "stream file" in Data Manager
+- **Activations** - you can configure multiple activations, they have to be of type `Ibexa` in [[= product_name =]] dashboard
+- **Client ID** and **Client secret** - Client ID and Secret pair is used to authenticate against Webhook endpoint, make sure it's random and secure
+- **Segment group identifier** - is the location where CDP data is imported
```

Comment:
```suggestion
| Setting | Description |
|--------|-------------|
| `account_number` | A [number](#account-number) obtained from the Account's settings in [[= product_name_cdp =]] dashboard. |
| `stream_id` | Stream ID generated when importing data from the stream file in Data Manager. |
| `activations` | Activation details. You can configure multiple activations. They have to be of type `Ibexa` in [[= product_name =]] dashboard. |
| `client_is` <br/> `client_secret`| Client credentials are used to authenticate against the Webhook endpoint. Make sure they are random and secure. |
| `segment_group_identifier` | A [location](#segment-group) to which CDP data is imported. |
```

Re. activations and credentials - Is an activation like an account at CDP? Where do the details come from? The dashboard? 
Same applies to stream id, I'd add short procedures similar to the ones below. How do I get to the Dada Manager etc.